graphix/gflow.py

```diff
@@ -195,8 +194,6 @@ def gflowaux(
             for i in range(len(x_col)):
                 node = node_order_col[col_permutation.index(i)]
                 g[non_out_node][node] = x_col[i]
-            for i in range(len(kernels)):
-                g[non_out_node][node_order_col[col_permutation.index(len(x_col) + i)]] = kernels[i]
             if meas_planes[non_out_node] in ["ZX", "YZ"]:
                 g[non_out_node][non_out_node] = sp.true
```

Comment:
Altough I personally like rotation-symmetric notation, "ZX" instead of "XZ", MBQC conventionally uses "XZ" and we are following this notation as in `pauli.Plane`. So, could you revert this change?